modules.d/01fips/fips.sh

```diff
@@ -47,7 +47,7 @@ mount_boot() {
         mkdir -p /boot
         fips_info "Mounting $boot as /boot"
         mount -oro "$boot" /boot || return 1
-    elif [ -d "$NEWROOT/boot" ]; then
+    elif ! ismounted /boot && [ -d "$NEWROOT/boot" ]; then
```

Comment:
What if /boot is not mounted as a separate partition but /boot directory exists as part of / (root) and it is not empty ?

Reply:
I understand this is super unlikely, but what I mean to ask is what if /boot is in the initramfs (e.g. added by "dracut --include  /boot/my_file /boot/my_file_in_initramfs")